NDTensors/src/diag/diagtensor.jl

```diff
@@ -33,6 +31,19 @@ function Array(T::DiagTensor{ElT,N}) where {ElT,N}
   return Array{ElT,N}(T)
 end
 
+function DiagonalArrays.diagview(T::NonuniformDiagTensor)
+  return data(T)
+end
+
+function DiagonalArrays.diagview(T::UniformDiagTensor)
+  return fill(getdiagindex(T, 1), diaglength(T))
+end
+
+## Should this go in dense.jl or here since its related to diag?
```

Comment:
Yes, I think this should go in `densetensor.jl`.

Reply:
I moved it to densetensor.jl, thanks!